README.md

```diff
@@ -50,45 +56,68 @@ wget https://raw.githubusercontent.com/biocore/q2-chemistree/master/q2_chemistre
+  --i-features sirius.mgf.qza \
+  --p-ppm-max 15 \
+  --p-profile orbitrap \
+  --p-n-jobs 1 \
+  --p-ionization-mode positive \
+  --p-java-flags "-Djava.io.tmpdir=/path-to-some-dir/ -Xms16G -Xmx64G" \
+  --o-fragmentation-trees FTs.qza
 ```
 
-First, we generate a contingency table with probabilities of presence of various molecular substructures in an MS1 feature (MS1 fingerprint):
+This generates a QIIME 2 artifact of type `SiriusFolder`. This contains fragmentation trees with candidate molecular formulas for each MS1 features detected in your experiment.
+**Note**: `/path-to-some-dir/` should be a directory where you have write permissions and sufficient storage space. We use -Xms16G and -Xmx64G as the minimum and maximum heap size for Java virtual machine (JVM). If left blank, q2-chemistree will use default JVM flags.
+
+Next, we retain top candidate per MS1 feature using [Zodiac](GitHublink) as follows:
 
 ```bash
-qiime chemistree fingerprint --p-sirius-path 'sirius-osx64-4.0.1/bin' \
+qiime chemistree rerank-molecular-formulas --p-sirius-path 'sirius-osx64-4.0.1/bin' \
   --i-features sirius.mgf.qza \
-  --p-profile orbitrap \
-  --p-ppm-max 15 \
+  --i-fragmentation-trees FTs.qza \
+  --p-zodiac-threshold 0.95 \
   --p-n-jobs 1 \
-  --o-collated-fingerprints collatedfp.qza
+  --p-java-flags "-Djava.io.tmpdir=/path-to-some-dir/ -Xms16G -Xmx64G" \
+  --o-molecular-formulas MFs.qza
 ```
+This produces a QIIME 2 artifact of type `ZodiacFolder` with top-ranked molecular formula for MS1 features. Now, we predict molecular substructures in each feature based on the molecular formulas. We use [CSI:FingerID](https://www.pnas.org/content/112/41/12580) for this purpose as follows:
 
-This generates a contingency table with probabilities of various molecular substructures (i.e. molecular fingerprints) within an MS1 feature of type `FeatureTable[Frequency]`.
+```bash
+qiime chemistree predict-fingerprints --p-sirius-path 'sirius-osx64-4.0.1/bin' \
+  --i-molecular-formulas MFs.qza \
+  --p-ppm-max 20 \
+  --p-n-jobs 1 \
+  --p-java-flags "-Djava.io.tmpdir=/path-to-some-dir/ -Xms16G -Xmx64G" \
+  --o-predicted-fingerprints FPs.qza
+  ```
+This gives us a QIIME 2 artifact of type `CSIFolder` that contains probabilities of molecular substructures (total 2936 molecular properties) within in each feature.
+We now generate a contingency table with these probabilities i.e. molecular fingerprints of MS1 features in our experiment. This is of type `FeatureTable[Frequency]`.
 
-Now, we generate the molecular tree using:
+```bash
+qiime chemistree collate-fingerprint --i-csi-result FPs.qza \
+--p-qc-properties \
+--o-collated-fingerprints collated_FPs_QC.qza
+```
+By default, we only use PUBCHEM fingerprints (total 489 molecular properties). Replacing the flag `--p-qc-properties` by `--p-no-qc-properties` retains all the molecular properties in the contingency table. This table is used to generate out hierarchy of molecules!

```

Comment:
Since the default is True (meaning the properties are going to be filtered), I would suggest that you rephrase this to say something along the lines of "Adding `--p-no-qc-properties` retains ...".